app/assets/javascripts/admin/addon/components/themes-grid.gjs

```diff
@@ -68,7 +55,36 @@ export default class ThemesGrid extends Component {
     ];
   }
 
+  @action
+  resortThemes() {
+    // Show default theme at the top of the list, we do the sort
+    // manually and cache it to make sure we don't reorder the list
+    // if the default theme changes.
+    this._cachedSortedThemes = this.args.themes.sort((a, b) => {
+      if (a.get("default")) {
+        return -1;
+      } else if (b.get("default")) {
+        return 1;
+      }
+      if (a.id < 0) {
+        return a.id;
+      }
+      if (b.id < 0) {
+        return -b.id;
+      }
+    });
+    this._sortPerformed = true;
+  }
+
+  @action
+  resetSortedThemes() {
+    this._cachedSortedThemes = null;
+    this._sortPerformed = null;
+  }
+
   <template>
+    <span {{didUpdate this.resetSortedThemes @themes}}></span>
```

Comment:
I don't like this...but didUpdate will not work if I put it on `AdminFilterControls`